src/popup/components/dashboard/Card.vue

```diff
@@ -42,8 +56,10 @@ export default {
     isBig: Boolean,
     clickable: Boolean,
     dense: Boolean,
+    closable: { type: String, default: null },
```

Comment:
Name 'closable' suggests that this is a boolean.
For the readability purpose I suggest to add another prop, eg.: `cardId` or `cardSlugName`.